modules/sandboxed-containers-about-sandboxing.adoc

```diff
@@ -0,0 +1,6 @@
+//Module included in the following assemblies:
+//
+// *understanding_sandboxed_containers.adoc
```

Comment:
Comment applies throughout PR: 
```
//Module included in the following assemblies:
//
// *understanding_sandboxed_containers.adoc
```

There should be a space after the `*` - it's a bulleted list

Reply:
We're also using the full relative path from the root of the ocp-docs repo, not just the file name here.

Reply:
Thanks, Jason! Might need to double check some of mine :)